sdk/python/lib/test/automation/test_local_workspace.py

```diff
@@ -750,6 +751,62 @@ def test_stack_lifecycle_inline_program(self):
         finally:
             stack.workspace.remove_stack(stack_name)
 
+    def test_stack_lifecycle_inline_program_remove_without_destroy(self):
```

Comment:
I think we need to adjust this test similar to the suggestions I made in the test for TypeScript.